packages/ui/src/components/DatePicker/PreviousDateRangePicker.tsx

```diff
@@ -224,11 +224,13 @@ const PreviousDateRangePickerImpl = ({
 	const [endTimeIsValid, setEndTimeIsValid] = useState<boolean>(true)
 
 	const menu = useMenu()
+	const open = menu.getState().open
```

Comment:
date picker choosing a custom date range is broken
![image](https://github.com/highlight/highlight/assets/1351531/273158f3-6509-4aa0-9b27-7469b5867040)

Reply:
Fixed in d6ed63c71b031b8e720838bc873218fc3c8bd45a